src/scss/General.scss

```diff
@@ -0,0 +1,10 @@
+@import '~scss/Variables';
+
+body {
+  margin: 0;
+}
+
+.page_wrapper {
+  position: relative;
+  top: $navbar-height;
```

Comment:
Going with a `top: $navbar-height` for now, but another option is to allow the navbar to sit on top of the page and just make the navbar transparent. What do you think, @simonzhow ?